tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -40,6 +40,14 @@ describe('mozilla-traffic-cop.js', function () {
             cop.init();
             expect(cop.verifyConfig).toHaveBeenCalled();
         });
+
+        it('should initialize if DNT helper function is not available', function() {
+            var dntBak = window._dntEnabled;
+            window._dntEnabled = false;
```

Comment:
I'll defer to you here. I'm not sure on best practices for wiping out an existing global function like this.

If we did `window._dntEnabled = sinon.stub().returns(undefined);`, wouldn't the `typeof` check still return `function`?